user/pages/03.Howtos/10.l3-high-availability/docs.en.md

```diff
@@ -20,13 +21,21 @@ taxonomy:
 
 * `keepalived` is an open source VRRP implementation
 
-## Considerations
+## Legacy vs New regions
+
+Our legacy regions (cbk, dbl) have a different SDN backed than our new regions (fes, dus2).
+Each SDN requires a dedicated `keepalived` configuration - both are presented below.
+
+[ui-tabs position="top-left" active="0" theme="lite"]
+[ui-tab title="Legacy regions (cbk, dbl)"]
```

Comment:
I'm not sure if this works well, at least in the rich diff mode, i could not see it working

Reply:
Do we still have a staging documentation system?

Reply:
Do you mean the tabs? They work very good. They are also used in the LBaaS page from Tutorials section:
https://docs.syseleven.de/syseleven-stack/en/tutorials/lbaas

Reply:
It's just that the common preview utilities don't show them correctly, so we cannot review in advance when they do not get parsed due to a typo or something like that.

Reply:
i could view preview locally, looks fine